manifests/m/Microsoft/PowerShell/Preview/7.4.0.5/Microsoft.PowerShell.Preview.installer.yaml

```diff
@@ -32,23 +32,5 @@ Installers:
   - Publisher: Microsoft Corporation
     DisplayName: PowerShell 7-preview-x86
   ElevationRequirement: elevatesSelf
-- Architecture: arm
-  MinimumOSVersion: 10.0.17763.0
-  Platform:
-  - Windows.Universal
-  InstallerType: msix
-  InstallerUrl: https://github.com/PowerShell/PowerShell/releases/download/v7.4.0-preview.4/PowerShell-7.4.0-preview.4-win.msixbundle
-  InstallerSha256: 3D8608AA1C8D345E7D992FD1EA56F537B9CF5270154732CA3C52ECB28A4158E9
-  SignatureSha256: F1E919A76A37E5BAB41D435570FD5954F71BE65818440202150B80B474314A25
-  PackageFamilyName: Microsoft.PowerShellPreview_8wekyb3d8bbwe
-- Architecture: arm64
-  MinimumOSVersion: 10.0.17763.0
-  Platform:
-  - Windows.Universal
-  InstallerType: msix
-  InstallerUrl: https://github.com/PowerShell/PowerShell/releases/download/v7.4.0-preview.4/PowerShell-7.4.0-preview.4-win.msixbundle
-  InstallerSha256: 3D8608AA1C8D345E7D992FD1EA56F537B9CF5270154732CA3C52ECB28A4158E9

```

Comment:
Hash needs to be updated

Reply:
The bot shouldn't have made the PR, as it's supposed to come from anamnavi or another employee.